### DISCUSSION TITLE
hosting capacity cigre_network

### DISCUSSION THREAD

**Discussion by rachidzarak** [e2nIEE/pandapower#1797] [Q&A]
Dear all,
How can i run hosting capacity with cigre network ?
net_cigre = nw.create_cigre_network_mv(with_der="all")

the results i've gotten for all iterations: 

installed | violation

0 | Line \n Overloading

Any help would be appreciated

**Comment by reconsumeralization** [comment]
Here is an example of how you can run hosting capacity analysis using the create_cigre_network_mv function in the pandapower library:

import pandapower as pp

# create a CIGRE MV grid with all DERs
net_cigre = pp.create_cigre_network_mv(with_der="all")

# run the hosting capacity analysis
pp.runpp(net_cigre)

# print the results
print("Installed capacity: ", net_cigre.res_trafo.loading_percent.sum())
print("Violation: ", net_cigre.res_line.loading_percent.max())
This will create a CIGRE medium voltage grid with all distributed energy resources (DERs) and run a hosting capacity analysis on it. The results will be printed, showing the total installed capacity and the maximum line loading percentage (violation).

You can also customize the input parameters for the create_cigre_network_mv function to create a specific type of CIGRE MV grid. For example, you can set with_der to "none" to create a grid without DERs, or to a list of specific DER types to include only those. You can also set other parameters such as the grid size and the percentage of DERs to include. For more information, you can refer to the documentation for the create_cigre_network_mv function.
